mlfinlab/labeling/raw_return.py

```diff
@@ -0,0 +1,43 @@
+"""
+Most basic form of labeling based on raw return of each observation relative to its previous value. User can specify
+percentage or logarithmic, numerical or categorical labels, and number of steps back to calculate the raw return from.
+"""
+
+import warnings
+import numpy as np
+
+
+def raw_return(price, binary=False, logarithmic=False, lookback=1):
+    """
+    Raw returns labeling method.
+
+    This is the most basic and ubiquitous labeling method used as a precursor to almost any kind of financial data
+    analysis or machine learning. User can specify percentage/logarithmic returns, numerical/binary labels, and
+    lookback period to calculate the return from.
+
+    :param price: (pd.Series or pd.DataFrame) Price data for one (Series) or multiple tickers (DataFrame).
+    :param binary: (bool) If False, will return numerical returns. If True, will return the sign of the raw return.
+                    False by default.
+    :param logarithmic: (bool) If False, will calculate percentage returns. If True, will calculate logarithmic returns.
+                        False by default.
+    :param lookback: (int) Number of ticks back to calculate each observation's return from. 1 by default. The first
```

Comment:
Perhaps worth replacing `Number of ticks back` with `Number of observations to look back` or `Lookback period`, as the input may not be in a tick form.